Script/AtomicNET/AtomicNETProject.json

```diff
@@ -23,6 +23,7 @@
             "name": "AtomicNET",
             "atomicNET" : true,
             "outputType" : "Library",
+            "defineConstants" : ["ATOMIC_PROFILING"],
```

Comment:
I mean the equivalent of ```${ATOMIC_RELEASE_OFF}```, for release builds the ATOMIC_PROFILING define should default off in C#, no need to worry about it now.